src/pudl/package_data/phmsagas/column_maps/yearly_distribution.csv

```diff
@@ -168,7 +168,7 @@ main_other_alt_4_to_8_in_miles,,,,over_4_thru_8_n8,ot2m_4,ot2m_4,ot2m_4,ot2m_4,o
 main_other_alt_8_to_12_in_miles,,,,over_8_thru_12_8,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,ot2m_5,,,,,,,,,,,,,,,
 main_other_alt_over_12_in_miles,,,,over_12_8,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,ot2m_6,,,,,,,,,,,,,,,
 main_other_alt_total_miles,,,,total_10,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,ot2mt,,,,,,,,,,,,,,,
-main_other_material_detail_miles,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail
+main_other_material_detail,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail,mmiles_other_material_detail
```

Comment:
these are not actually miles, its a note about what the heck the other material is... will need to figure out where to put this in the million tables